README.md

```diff
@@ -44,9 +44,35 @@ Detect secrets...........................................................Passed
 ```
 
 
+## Run your first audit
+
+The detect-secrets tool may discover a secret in your repository that you want to exclude because it's a false positive. It may also find lots of legitimate secrets that you need to confirm.
+
+If you receive an error message regarding your secrets baseline file, it will most likely be because you didn't follow the [usage](https://github.com/alphagov/gds-pre-commit#usage) and run your first audit guidelines.
+
+### Ensure your repo contains no secrets and ignores false positives
+[Run an audit](https://github.com/Yelp/detect-secrets#auditing-a-baseline) against your `.secrets.baseline` file before your run your first commit after installation.
+
+Once you have run a `scan` to create a `.secrets.baseline` file as mentioned in the [Usage section](https://github.com/alphagov/gds-pre-commit#usage) above, run the following command:
+
+```shell
+detect-secrets audit /<path-to-your-git-repo>/.secrets.baseline
+```
+
+False positives can also be ignored using [inline comments](https://github.com/Yelp/detect-secrets#inline-allowlisting), but running an audit is the preferred method.
+
+### Check in your .secrets.baseline file
+
+Once you've run your first audit of a repository, check the `.secrets.baseline` file in. This way, your colleagues/collaborators won't have to run it themselves.
+
```

Comment:
checked in = commited?
If it is 'check in' I think we should change this sentence to: "check **in**the `.secrets.baseline` file"